pkg/hintrunner/zero/zerohint.go

```diff
@@ -24,7 +24,11 @@ func (hint *GenericZeroHinter) Execute(vm *VM.VirtualMachine, ctx *hinter.HintRu
 }
 
 func GetZeroHints(cairoZeroJson *zero.ZeroProgram) (map[uint64][]hinter.Hinter, error) {
-	hints := make(map[uint64][]hinter.Hinter)
+	numHints := 0
+	for _, rawHints := range cairoZeroJson.Hints {
+		numHints += len(rawHints)
+	}
+	hints := make(map[uint64][]hinter.Hinter, numHints)
```

Comment:
couldn't you just pass the length of cairoZeroJson.Hints to initialize the map? If I understand correctly, that second parameter for `make` should indicate the length of the mapping, which would be the amount of entries, but `numHints` in this case is the total amount of hints. Am I mistaken?

Reply:
yeah, it's not going the be just the length of `cairoZeroJson.Hints`. Since the hint can occur in multiple places, we are interested in the size of the array associated with that hint.